src/molecules/AccountIndicator/account_indicator.module.scss

```diff
@@ -8,7 +8,6 @@ $default-spacing: rem-calc(16px);
 .indicator {
   width: auto;
   display: flex;
-  justify-content: end;
```

Comment:
Just double checking this is right, because this field is not deprecated according to the spec: https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

Reply:
Agree that the behavior is wrong, just double checking rational.

Reply:
Not deprecated but "mixed support", which is enough for Chrome to show warnings about it. In this case, it's not applicable to the elements it's styling, so I think we need @bradwheel to weigh in on whether we need this to be `flex-end` or let it fall back to the default.